.github/workflows/deploy-swagger-ui.yaml

```
@@ -0,0 +1,55 @@
+name: Deploy Swagger UI
+
+on:
+  push:
+    branches-ignore:
+      - '*'
+    # branches:
+      # - main
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
```

```suggestion
jobs:
  build-and-push:
    if: {{ false }}
    runs-on: ubuntu-latest
```

https://stackoverflow.com/questions/68227475/can-you-pause-disable-a-github-action

링크에서는 이렇게 하면 스킵이 된다고 되어있기는 한데 제가 시도해본 방법은 아니라서 이미 확인한 내용이실 수도 있습니다.. 확인 한번 다시 해보시고 해보시면 좋을 것 같아요 !

그게 아니라면 지금 당장에는 워크플로우가 새롭게 작성되고 다른 실행될 워크플로우가 없으니 비활성화 해둬도 괜찮을 것 같습니다 !
https://docs.github.com/ko/enterprise-cloud@latest/actions/managing-workflow-runs-and-deployments/managing-workflow-runs/disabling-and-enabling-a-workflow

---

오! job 실행 조건을 false로 설정하니까 실행이 안되네요!! 감사합니다 👍